app/hooks/use-realtime-price.ts

```diff
@@ -0,0 +1,25 @@
+import { WatchQueryFetchPolicy } from "@apollo/client"
+import { useDisplayCurrencyQuery, useRealtimePriceQuery } from "@app/graphql/generated"
+import { useIsAuthed } from "@app/graphql/is-authed-context"
+
+export const useRealtimePriceWrapper = (
+  { fetchPolicy }: { fetchPolicy: WatchQueryFetchPolicy } = {
+    fetchPolicy: "network-only",
```

Comment:
I dont think "network-only" is the desired fetch policy here. I believe this will cause a network request to be made on each rerender. You could instead use a lazy query or just refetch when desired.

Reply:
earlier comment on this here: https://github.com/GaloyMoney/galoy-mobile/pull/1270#discussion_r1117984415

Reply:
only if the component unmount and remount. ie screen closed and reopen

Reply:
I misunderstood how this hook worked. You can ignore my comments.